web_frontend/cloudscheduler/csv2/templates/csv2/jobs.html

```diff
@@ -0,0 +1,199 @@
+<html>
+
+	<head>
+        {% load static %}
+
+        <link rel="stylesheet" type="text/css" href="{% static "csv2/common.css" %}?{{version}}">
+        <link rel="stylesheet" type="text/css" href="{% static "csv2/jobs.css" %}?{{version}}">
+
+	<!-- <meta http-equiv="refresh" content="30"> -->
+	</head>
+
+
+	<body>
+		<h2>Jobs for group=<i>{{ jobs_list.0.group_name }}</i>
```

Comment:
Did you test what happens if there is no jobs here? Does this error out at the rendering step?